Framework/PythonInterface/plugins/algorithms/LoadEXED.py

```diff
@@ -85,7 +85,7 @@ def PyExec(self):
         # nbins=int(parms_dict['NTC'])
         xdata = np.array(det_tbc)
         xdata_mon = np.linspace(xdata[0], xdata[-1], len(xdata))
-        ydata = data.astype(np.float)
```

Comment:
Can I ask about these `numpy` changes? They don't seem related to `matplotlib`?

Reply:
They popped up whilst working on matplotlib, and were throwing warnings, technically not related, but still an issue in our conda packages.

Reply:
Ok. Do we know if there are tests against the current versions of `numpy` such that these will still work?

Reply:
According to the advice from the warning they mean the same thing. np.float == float.